_ml-commons-plugin/model-serving-framework.md

```diff
@@ -139,9 +152,10 @@ POST /_plugins/_ml/models/WWQI44MBbzI2oUKAvNUt/_load
 }
 ```
 
-#### Sample Response
 
-All models load asynchronously. Therefore, the load API responds a new `task_id` based on the load, and returns a `status` for the task.
+#### Sample response
+
+All models load asynchronously. Therefore, the load API responds a new `task_id` based on the load and responds with a new `status` for the task.
```

Comment:
"Therefore, the load API responds with a new `task_id` based on ..."
Add "with" ?